genon/preprocessor/facade/legacy/BOK_적재용_규정.py

```
@@ -0,0 +1,1974 @@
+        # ================================================================
+        # 2.5단계: 너무 긴 청크는 분할
+        # ================================================================
+        if self.max_tokens > 0:
+            for i in range(len(sections_with_text)):
+                text, items, h_infos, h_short = sections_with_text[i]
+                token_count = self._count_tokens(text)
+                if token_count < self.max_tokens:
+                    continue
+
+                # caption 및 table 내 그림은 같은 섹션에 있도록 조정
+                items_group=[[(item, info, short)] for item, info, short in zip(items, h_infos, h_short)]
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)